plugins/tech-insights-common/api-report.md

```diff
@@ -25,6 +25,7 @@ export interface CheckResponse {
 export type CheckResult = {
   facts: FactResponse;
   check: CheckResponse;
+  result: any;
```

Comment:
I was pondering on this last night whether or not to actually introduce type params to this shape as well. I think it's probably not a good idea since it'll end up propagating to too many code paths across all usage sites. `JsonValue` could be good (maybe aliased as Serializable?)

Reply:
I agree JsonValue is a good fit for this case, thanks for suggesting it :) Regarding type params here @Xantier, out of curiosity, what benefit you see if we do introduce it here?

Reply:
We'd get shareable type safety on the FE side as well via those common type definitions. I think it will be too cumbersome to carry that type param across all call sites for now though